src/Concordium/Client/GRPC2.hs

```diff
@@ -0,0 +1,3222 @@
+        (Just cred, Nothing, Nothing) -> CredRegID $ fromProto cred
+        (Nothing, Just addr, Nothing) -> AccAddress $ fromProto addr
+        (Nothing, Nothing, Just accIdx) -> AccIndex $ fromProto accIdx
+        _ -> error "nope"
+
+instance ToProto AccountIdentifierInput where
+    type Output AccountIdentifierInput = Proto.AccountIdentifierInput
+    toProto = \case
+        CredRegID cred -> Proto.make $ ProtoFields.credId .= toProto cred
+        AccAddress addr -> Proto.make $ ProtoFields.address .= toProto addr
+        AccIndex accIdx -> Proto.make $ ProtoFields.accountIndex .= toProto accIdx
+
+instance FromProto Proto.SequenceNumber where
+    type Output' Proto.SequenceNumber = Nonce
+    fromProto = Nonce . deMkWord64
+
+instance FromProto Proto.ReleaseSchedule where
+    type Output' Proto.ReleaseSchedule = AccountReleaseSummary
+    fromProto ars =
+        AccountReleaseSummary
+            { releaseTotal = fromProto $ ars ^. ProtoFields.total
+            , releaseSchedule = fromProto <$> (ars ^. ProtoFields.schedules)
+            }
+
+instance FromProto Proto.Timestamp where
+    type Output' Proto.Timestamp = (Timestamp, UTCTime)
+    fromProto t = (Timestamp time, utcTime)
+      where
+        time = fromIntegral $ t ^. ProtoFields.value
+        utcTime = posixSecondsToUTCTime (fromIntegral time / 1_000.0)
+
+instance FromProto Proto.TransactionHash where
+    type Output' Proto.TransactionHash = TransactionHashV0
+    fromProto = deMkSerialize
+
+instance FromProto Proto.Release where
+    type Output' Proto.Release = ScheduledRelease
+    fromProto r =
+        ScheduledRelease
+            { releaseTimestamp = fst . fromProto $ r ^. ProtoFields.timestamp
+            , releaseAmount = fromProto $ r ^. ProtoFields.amount
+            , releaseTransactions = fromProto <$> (r ^. ProtoFields.transactions)
+            }
+
+instance FromProto Proto.AccountThreshold where
+    type Output' Proto.AccountThreshold = AccountThreshold
+    fromProto = AccountThreshold . deMkWord8
+
+instance FromProto Proto.EncryptedAmount where
+    type Output' Proto.EncryptedAmount = EncryptedAmount
+    fromProto = deMkSerialize
+
+instance FromProto Proto.EncryptedBalance where
+    type Output' Proto.EncryptedBalance = AccountEncryptedAmount
+    fromProto eb =
+        AccountEncryptedAmount
+            { _selfAmount = fromProto $ eb ^. ProtoFields.selfAmount
+            , _startIndex = EncryptedAmountAggIndex . fromIntegral $ eb ^. ProtoFields.startIndex
+            , _aggregatedAmount =
+                case (fromProto <$> eb ^? ProtoFields.aggregatedAmount, fromIntegral <$> eb ^? ProtoFields.numAggregated) of
+                    (Just aggAmt, Just numAgg) -> Just (aggAmt, numAgg)
+                    (Nothing, Nothing) -> Nothing
+                    _ -> error "oops"
+            , _incomingEncryptedAmounts = Empty -- FIXME: Field is not included in other direction. Should it be?
```

Comment:
What do you mean it is not included in the other direction? Most accounts will have this empty, so you need to find one which has some incoming encrypted transfers. Just make one using the client to have an account to test with.